### ISSUE TITLE
Insertion of NuGet 5.5.0-preview.1.6293

### ISSUE THREAD

**Issue by zkat** [NuGet/Home#8802]
VS:
* [x] Insertion PR - https://devdiv.visualstudio.com/DevDiv/_git/VS/pullrequest/213394

.NET Core (master):
* [ ] SDK - TODO
* [ ] CLI - TODO
* [ ] Toolset - TODO

**Comment by nkolev92** [comment]
Blocked waiting on a branch for the 5.5/16.5 equivalent SDK release.

**Comment by nkolev92** [comment]
No P1 of SDK. 

This will eventually get superseded by a P2 insertion.
